utils/utils/src/main/scala/pl/touk/nussknacker/engine/util/TimestampUtils.scala

```diff
@@ -0,0 +1,29 @@
+package pl.touk.nussknacker.engine.util
+
+import java.sql.Timestamp
```

Comment:
What sql do you have in your code so you have to use SQL types?

Reply:
it's just an IntelliJ miss-import, my bad